modules/virtualization/microvm/appvm.nix

```diff
@@ -68,6 +74,8 @@
             qemu.extraArgs = lib.optionals (config.nixpkgs.hostPlatform.system == "x86_64-linux") [
               "-bios"
               "${pkgs.qboot}/bios.bin"
+              "-M"
+              "microvm,pcie=on"
```

Comment:
Shouldn't this also have rest orf the options from the microvm.nix?
https://github.com/astro/microvm.nix/blob/main/lib/runners/qemu.nix#L35

Reply:
Like @nesteroff has done here: https://github.com/tiiuae/ghaf/pull/256/files#diff-b7c4d8b1828ec4e55c3b85b348f4c180560550100c49b79a826d4cacfd1afaedR78

Reply:
Actually, It looks like it preserves previously set machine props. For example, it doesn't complain about missing pcie when I pass this:
```
-M microvm,pcie=on \                                                                                                                                                                             
-M microvm,rtc=off 
```
It is still a hack though.

Reply:
I agree with you, it should be better indicated which props are active. However adding "all props" to multiple places is not good solution as then modifying some single prop gets complicated.